[2024-12-04 23:04:33] INFO - train: Starting training process...
[2024-12-04 23:04:33] INFO - train: Loading configurations...
[2024-12-04 23:04:33] INFO - train: Using device: cuda
[2024-12-04 23:04:33] INFO - train: Creating data loaders...
[2024-12-04 23:05:14] INFO - train: Setting up models and trainer...
[2024-12-04 23:05:14] INFO - train: Initializing distillation training...
[2024-12-04 23:05:18] INFO - train: Starting training...
[2024-12-04 23:05:19] ERROR - train: An error occurred: 'logits'
Traceback (most recent call last):
  File "/nas2/jieui/class/deepnlp/project/scripts/train.py", line 97, in main
    trainer.train()
  File "/nas2/jieui/class/deepnlp/project/src/trainers/base_trainer.py", line 74, in train
    loss = self._training_step(batch)
  File "/nas2/jieui/class/deepnlp/project/src/trainers/distillation_trainer.py", line 83, in _training_step
    F.log_softmax(student_outputs['logits']  / temperature, dim=-1),
KeyError: 'logits'
